options/locale/locale_en-US.ini

```diff
@@ -1261,41 +1261,41 @@ users.activated = Activated
 users.admin = Admin
 users.repos = Repos
 users.created = Created
-users.last_login = Last Login
-users.never_login = Never Login
+users.last_login = Last Sign-In
+users.never_login = Never Signed-In
 users.send_register_notify = Send Registration Notification To User
 users.new_success = The account '%s' has been created.
 users.edit = Edit
 users.auth_source = Authentication Source
 users.local = Local
-users.auth_login_name = Authentication Login Name
+users.auth_login_name = Authentication Sign-In Name
 users.password_helper = Leave it empty to remain unchanged.
 users.update_profile_success = Account profile has been updated.
 users.edit_account = Edit Account
 users.max_repo_creation = Maximum Repository Creation Limit
 users.max_repo_creation_desc = (Set -1 to use global default limit)
 users.is_activated =  Account activated
-users.prohibit_login = Login disabled
+users.prohibit_login = Sign-in disabled
 users.is_admin = Administrator permissions
 users.allow_git_hook = Allowed to create git hooks
 users.allow_import_local = Allowed to import local repositories
 users.allow_create_organization = Allowed to create organizations
 users.update_profile = Update Account Profile
 users.delete_account = Delete Account
-users.still_own_repo = This user still owns one or more repositories. These repositories need to be deleted or transferred first.
+users.still_own_repo = This user still owns one or more repositories. These repositories need to be deleted or be transferred first.
 users.still_has_org = This user is still is a member of one or more organizations. This user needs to leave or delete them first.
-users.deletion_success = Account deleted successfully.
+users.deletion_success = The account has been deleted.
 
 orgs.org_manage_panel = Organization Management
 orgs.name = Name
 orgs.teams = Teams
 orgs.members = Members
-orgs.new_orga = Create Organization
+orgs.new_orga = New Organization
 
 repos.repo_manage_panel = Repository Management
 repos.owner = Owner
 repos.name = Name
-repos.private = Private
```

Comment:
I think this should be left as private

Reply:
I wanted to remove a layer of cognitive indirection ('Private' means 'Hides Repository'), but it's your call. I'll update my PR in a few minutes.

Reply:
@lafriks : can you update your review status? I think I've correctly reverted to 'private' in my PR.